test/stepDefinitions/caseDetailsSteps.js

```diff
@@ -97,9 +97,12 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
   });
 
   When(/^I click on its first accordion on the '(.*)' tab$/, async function (tabName) {
-    await caseDetailsPage.clickTab(tabName);
-    await caseDetailsPage.clickFirstAccordian();
+      await caseDetailsPage.clickTab(tabName);
+      await caseDetailsPage.clickFirstAccordian();
   });
 
+  When(/^I progress to the next event which contains read-only value of the selected fixed list value$/, async function() {
```

Comment:
there is a mismatch between that this method is doing and the step description. For example there's nothing regarding read only or fixed lists in the method implementation.
We should make the step description more generic. If we do that we would also improve reusability